addon/components/base-chart-component.js

```diff
@@ -85,7 +85,7 @@ export default Component.extend({
 
         // NOTE: THIS BEING HERE IS ASSUMING ALL CHARTS WILL USE ALL OF THESE PROPERTIES. MAY NOT BE A VALID ASSUMPTION
         // REQUIRED: group, dimension, xAxis.domain unless !isChartAvailable
-        if (!this.get('group') || !this.get('group.0.all') || !this.get('dimension')) {
+        if (!this.get('group') || /* !this.get('group.0.all') ||*/ !this.get('dimension')) {
```

Comment:
dead code that can be removed?

Reply:
Yep, you're right. For clarity: ALL charts MUST have a group and a dimension, so those are valid assumptions. However, not all charts' group is an array (most notably the pie chart), which is also addressed in the `didReceiveAttrs()` function; therefore, `this.get('group')[0]` will not always be truthy. Also, all charts do not need xAxis.domain, so I removed that part of the comment.